### ISSUE TITLE
为什么不能用这个插件直接请求deepL的接口？

### ISSUE THREAD

**Issue by mortalBibo** [missuo/bob-plugin-deeplx#8]
首先感谢作者的无私奉献，小白想请教一个问题，本地docker起一个deeplX的go语言服务，是为了接收要翻译的内容，并伪装成正常请求去请求deepL的API，相当于这个插件去请求deeplX，deeplX通过一些逻辑处理，再去请求deepL的官方API，那为什么不直接用这个插件去实现deeplX的处理逻辑呢？这样就可以直接去请求deepL的官方接口了，Go写的逻辑JS不是一样能实现

**Comment by missuo** [comment]
The original intention of doing the DeepLX project is not only for the Bob plugin, but also to be able to integrate any translation tool. So it is not integrated into the plugin. As far as I know, someone has already written an integrated version of the Bob plugin. If you are interested, you can take a look.

In addition, there is another issue built into the Bob plugin, which is that when your IP is blocked, it seems that you can only unblock it by starting the proxy software through Mac. However, with the Bob plugin I wrote, you can simply change the API URL to make it work normally.

**Comment by mortalBibo** [comment]
> The original intention of doing the DeepLX project is not only for the Bob plugin, but also to be able to integrate any translation tool. So it is not integrated into the plugin. As far as I know, someone has already written an integrated version of the Bob plugin. If you are interested, you can take a look.
> 
> In addition, there is another issue built into the Bob plugin, which is that when your IP is blocked, it seems that you can only unblock it by starting the proxy software through Mac. However, with the Bob plugin I wrote, you can simply change the API URL to make it work normally.

插件改变API URL不解决本质问题吧，如果是本地部署的deeplX的IP被封，有什么办法解决吗？还是只能通过代理软件去切换
非常感谢您的回复，祝生活愉快～
